src/test/scala/mesosphere/marathon/state/ContainerTest.scala

```diff
@@ -152,7 +151,7 @@ class ContainerTest extends MarathonSpec with Matchers {
     val proto = ContainerSerializer.toProto(f.container)
     assert(mesos.ContainerInfo.Type.DOCKER == proto.getType)
     assert("group/image" == proto.getDocker.getImage)
-    assert(f.container.volumes == proto.getVolumesList.asScala.map(Volume(_)))
+    assert(f.container.volumes == proto.getVolumesList.asScala.map(Volume.fromProto(_)))
```

Comment:
@kolloch serialization was already split, I just brought it into state vs. moved it all to api.serialization. I DID get an OK from a matthias via slack or something before I did this.